src/typings.ts

```diff
@@ -107,6 +107,7 @@ export interface Amount {
 // EVENTS
 export interface EventFilterOptions {
   type?: string
+  contractType?: string
```

Comment:
You could enforce the usage of valid contract types here. Either using string literals
```ts
type ValidContractTypes =  'CurrencyNetwork' | 'Exchange' | 'UnwETH' | 'Token'
```
or using an enum.